go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/docker/go-units v0.4.0
 	github.com/go-openapi/runtime v0.25.0
 	github.com/go-openapi/strfmt v0.21.3
+	github.com/signadot/go-sdk v0.3.7-0.20230310214135-3f47ce384c70
```

Comment:
Result of `go get github.com/signadot/go-sdk@main` that points to the latest main branch which at the time of executing the command is https://github.com/signadot/cli/commit/eea80de6781fbcbdc06f64aab5d7e9484c1a787.

I'm not sure what the version here maps to `v0.3.7-0.20230310214135-3f47ce384c70`.  But I have verified that it works. If you know, pls lmk as well :)

Reply:
My bad. That was the latest commit in the cli repo. The correct commit is https://github.com/signadot/go-sdk/commit/3f47ce384c70b41339c9a9665a6c5940e416277b. So the SHA is a match. 👍